src/lib/Runner.ts

```diff
@@ -190,6 +191,60 @@ export class Runner {
     return state.paused ? { isPaused: true, reason: state.reason } : { isPaused: false };
   };
 
+  async sendBannerMessage(
+    message: string,
+    messageType: IMessageState['messageType'],
+    user: ISessionUser,
+  ) {
+    await MessageStateTransition.create<MessageStateTransition>({
+      senderAaid: user.aaid,
+      messageExists: true,
+      message,
+      messageType,
+    });
+  }
+
+  async removeBannerMessage(user: ISessionUser) {
+    await MessageStateTransition.create<MessageStateTransition>({
```

Comment:
just to be sure - it will remove or hide the banner message?

Reply:
This will remove the banner message, as the structure is a transition between the a banner message existing and not existing. There would be no way of re-showing the message without creating a new one.